javascript/pull-view.js

```diff
@@ -29,13 +29,14 @@
         this.$el.addClass('unimportant-repo');
       }
 
-      if (this.model.comment.get('thumbsup')) {
-        this.$el.addClass("thumbsup");
+      if (this.model.comment.get('thumbsup') || this.model.reviewComment.get('approved')) {
```

Comment:
Both your original code and my suggestion are logically the same. Agree that both classes getting applied would be bad, but I don’t think this can happen, unless it’s somehow looping over the code multiple times for one pull request.